configure.ac

```diff
@@ -3,7 +3,7 @@
     AC_CONFIG_HEADERS([config.h])
     AC_CONFIG_SRCDIR([src/suricata.c])
     AC_CONFIG_MACRO_DIR(m4)
-    AM_INIT_AUTOMAKE([tar-ustar])
+    AM_INIT_AUTOMAKE([tar-ustar], [subdir-objects])
```

Comment:
This should be:
```
     AM_INIT_AUTOMAKE([tar-ustar subdir-objects])
```
This will eliminate the following errors during autogen:
```
src/Makefile.am:573: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:573: but option 'subdir-objects' is disabled
```

Reply:
Ok, coming in fuzz-v23